packages/firebase_auth/firebase_auth_platform_interface/test/method_channel_tests/method_channel_firebase_auth_test.dart

```diff
@@ -407,18 +404,18 @@ void main() {
         expect(result, isA<UserCredentialPlatform>());
         expect(result.user, isA<UserPlatform>());
 
-        final userResult = result.user;
+        final userResult = result.user!;
         expect(userResult.uid, isA<String>());
         expect(userResult.email, isNull);
         expect(userResult.isAnonymous, isTrue);
-        expect(userResult.uid, equals(auth.currentUser.uid));
+        expect(userResult.uid, equals(auth.currentUser!.uid));
 
         final additionalUserInfo = result.additionalUserInfo;
-        expect(additionalUserInfo, isA<Object>());
+        expect(additionalUserInfo, isNull);
```

Comment:
`isA<Object>` is always true. Using `isNull` since that is the current behaviour, but could be wrong